lib/systems/include/Controllers/SimpleController.h

```diff
@@ -28,8 +28,10 @@ class TorqueControllerSimple : public Controller
         writeout_.command = BaseControllerParams::TC_COMMAND_NO_TORQUE;
         writeout_.ready = true;
     }
-
+    /// @brief calculates torqe output based off max torque and simple torque scaling metric defined in constructor and adjusts writeout_ accordingly
+    /// @param pedalsData controller calculates acceleration request from the pedals based off of this data
     void tick(const PedalsSystemData_s &pedalsData);
+    /// @note refer to parent class
```

Comment:
I'd prefer this specifically say "refer to parent class for function documentation". Refer to parent class for what? Where in parent class?